include/macros.inc

```diff
@@ -1,27 +1,27 @@
 /*
 Code sections:
-	.text0:	0x00000100	0x80004000	0x800064E0  # init
-	.text1:	0x000025E0	0x80007020	0x8016A6E0  # text
+    .init   80003100 - 80005598 : 002498
+    .text   800055A0 - 800D2F8C : 0CD9EC
 Data sections:
-	.data0:	0x00165CA0	0x800064E0	0x800069A0  # extab_
-	.data1:	0x00166160	0x800069A0	0x80007020  # extabindex_
-	.data2:	0x001667E0	0x8016A6E0	0x8016A740  # ctors
-	.data3:	0x00166840	0x8016A740	0x8016A760  # dtors
-	.data4:	0x00166860	0x8016A760	0x8016F7A0  # rodata
-	.data5:	0x0016B8A0	0x8016F7A0	0x801A0CC0  # data
-	.data6:	0x0019CDC0	0x8025C6C0	0x8025CFE0  # sdata
-	.data7:	0x0019D6E0	0x8025DD40	0x8025EFA0  # sdata2
-BSS sections:
-	.bss:	0x00000000	0x801A0CC0	0x8025C6C0  # bss
-	.bss:	0x00000000	0x8025CFE0	0x8025DD40  # sbss
-	.bss:	0x00000000	0x8025EFA0	0x8025EFB8  # sbss2
-Entry Point: 0x8000403C
+    .ctors  800D2FA0 - 800D2FA8 : 000008
+    .dtors  800D2FC0 - 800D2FC8 : 000008
+    .rodata 800D2FE0 - 800D3710 : 000730
+    .data   800D3720 - 800F3ED8 : 0207B8
+    .sdata  80134CE0 - 80135579 : 000899
+    .sdata2 80135D00 - 80136420 : 000720
+BSS section:
+    .bss    800F3EE0 - 80134CD8 : 040DF8
+    .sbss   80135580 - 80135CFD : 00077D
+    .sbss2  80136420 - 80136420 : 000000
+Entry Point: 0x80003154
 */
+
```

Comment:
yeah I think we can get rid of this, looks like dtk have its own macros.inc

Reply:
Oh, I actually just mean the comment on top of the file. But on second thought, if this file is unused by your rewrite, I think we should revert it so that oot-vc's asm is easier to get working again

Reply:
ooh lol, right I'll revert it then, I misunderstood